workflow.qmd

```diff
@@ -2,25 +2,25 @@
 title: "U.S. Greenhouse Gas Center: Data Workflow Documentation"
 ---
 
-Welcome to the [U.S. Greenhouse Gas (GHG) Center: Data Workflow Documentation](https://earth.gov/ghgcenter), your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data workflow documentation, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
```

Comment:
Recommend replacing "Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data workflow documentation, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions. Discover the journey of each dataset with our detailed data workflows. Gain insights into how we handle and process data, from acquisition to analysis. Explore examples of data workflows for transparency and understanding of our rigorous processes." with:

Welcome to the U.S. Greenhouse Gas (GHG) Center data workflow documentation. View data flow diagrams to discover the journey of each dataset from acquisition into the US GHG Center. 

Data flow diagrams are grouped topically and labeled by dataset name. The dataset product type (model output, satellite observation, etc.) is noted next to each. Click on a dataset name to view the data flow diagram for that dataset, which summarizes the process followed to bring the dataset into the US GHG Center.